geomesa-convert/geomesa-convert-avro/src/test/scala/org/locationtech/geomesa/convert/avro/AvroSimpleFeatureConverterTest.scala

```diff
@@ -1,95 +0,0 @@
-/***********************************************************************
- * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Apache License, Version 2.0
- * which accompanies this distribution and is available at
- * http://www.opensource.org/licenses/apache2.0.php.
- ***********************************************************************/
-
-package org.locationtech.geomesa.convert.avro
-
-import java.io.ByteArrayInputStream
-
-import com.typesafe.config.ConfigFactory
-import org.junit.runner.RunWith
-import org.locationtech.geomesa.convert2.SimpleFeatureConverter
-import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
-import org.specs2.mutable.Specification
-import org.specs2.runner.JUnitRunner
-
-@RunWith(classOf[JUnitRunner])
-class AvroSimpleFeatureConverterTest extends Specification with AvroUtils {
```

Comment:
Le sigh, if you figure out how to get your IDE to use git mv, that'd be awesome:).

Reply:
I looked into git mv, but ultimately it doesn't make a difference. it appears the only way to preserve a file is to make it two separate commits.